common/json-schema-to-grammar.cpp

```diff
@@ -16,7 +16,7 @@ static std::string join(Iterator begin, Iterator end, const std::string & separa
 
 static std::string repeat(const std::string & str, size_t n);
 
-static std::string build_repetition(const std::string & item_rule, int min_items, int max_items, const std::string & separator_rule = "") {
+static std::string build_repetition(const std::string & item_rule, const int min_items, int max_items, const std::string & separator_rule = "") {
```

Comment:
If you're changing `min_items` to `const`, is it worth adding `const` to `max_items` as well?

Reply:
Dropped unintended change, thanks!